src/ui/server/helpers/mappings/map-currencies/as-select-options/index.ts

```diff
@@ -0,0 +1,37 @@
+import getSupportedCurrencies from '../map-supported-currencies';
+import mapSelectOption from '../../map-select-option';
+import sortArrayAlphabetically from '../../../sort-array-alphabetically';
+import { Currency } from '../../../../../types';
+
+/**
+ * mapCurrenciesAsSelectOptions
+ * Map all currencies into the required structure for GOV select component.
+ * @param {Array} Array of currency objects
+ * @param {String} Selected currency
+ * @returns {Array} Array of mapped currencies
+ */
+const mapCurrenciesAsSelectOptions = (currencies: Array<Currency>, selectedValue?: string) => {
+  const supportedCurrencies = getSupportedCurrencies(currencies);
+
+  const mapped = supportedCurrencies.map(({ name, isoCode }) => mapSelectOption(name, isoCode, true, selectedValue));
```

Comment:
should we name a variable as true so we know exactly what is being set as true without going to the function?

Reply:
This has been around for ages 😄  but good shout, have updated